src/site/stages/build/process-cms-exports/transformers/node-event.js

```diff
@@ -19,6 +20,15 @@ function toUtc(timeString) {
   return time.format('YYYY-MM-DD kk:mm:ss [UTC]');
 }
 
+function toUtcTz(timeString) {
```

Comment:
Could this just be combined with `toUtc`?

```js
function toUtc(timeString, withExplicitUtc = true) {
  ...
  return time.format(
    withExplicitUtc ? 'YYYY-MM-DD kk:mm:ss [UTC]' : 'YYYY-MM-DD[T]kk:mm:ss'
  );
}
```

Reply:
Yes, I was going to refactor it but in support now. I'll refactor when I get back to it